third-party/realdds/doc/DDS ICD.md

```diff
@@ -0,0 +1,147 @@
+# realdds Topics
+
+There are 2 topic types in the realdds domain
+* **flexible** - Carries a buffer of data that can be JSON, CBOR or other user custom data
+See https://github.com/IntelRealSense/librealsense/blob/dds/third-party/realdds/include/realdds/topics/flexible/flexible.idl
+* **image** - A buffer of data with some extra fields that describe the image
+See https://github.com/IntelRealSense/librealsense/blob/dds/third-party/realdds/include/realdds/topics/image/image.idl
+
+There are 4 main topics, consisted of these 2 types.
+* **realdds/device-info** of type flexible - The server sends a sample of this topic to inform readers of a connected camera.
+  Some of the details included are the camera model and serial number, this information will be used in the other topic names to ease communication with verious cameras in the network.
+* **realdds/\<model\>/\<serial\>/notification** of type flexible - Notifications are sent from the camera server to the camera user.
+  Some notificaitons are being sent at discovery to convey the full device information, e.g streams, supported options, etc...
+  Other notifications are sent as needed, mostly as reply to a control message
+* **realdds/\<model\>/\<serial\>/control** of type flexible - Controls are commands from the camera user to the camera, e.g start streaming, set option value etc...
+* **realdds/\<model\>/\<serial\>/<stream name>** of type image - Frames of data of the appropriate stream from the camera server to the user.
+
+# Messages
+
+## device-info
+
+On the camera user side, `realdds::dds_device_watcher` can be used to listen to incoming device-info messages.
```

Comment:
syntax: camera's user side

Reply:
Maybe you can write: 
The camera's user side should use `realdds::dds_device_watcher` to listen to incoming device-info messages.
Didn't you have some examples for server and client? It would be great to understand the use.

Reply:
I didn't use *should* on purpose. The main focus of this document is to lay the topics structure so even someone not using this library, or librealsense at all, would be able to connect (like Yoram's team is planned to do).

The use-case documents and design diagrams would indeed help to understand how to use realdds objects.